connectors/salesforce/src/main/java/com/vmware/connectors/salesforce/SalesforceController.java

```diff
@@ -210,7 +191,7 @@ public SalesforceController(AsyncRestOperations rest, CardTextAccessor cardTextA
         headers.set(AUTHORIZATION, sfAuth);
         headers.set(CONTENT_TYPE, APPLICATION_JSON_VALUE);
 
-        Map<String, String> bodyMap = new HashMap<>();
+        final Map<String, String> bodyMap = new ConcurrentHashMap<>();
```

Comment:
Controversial again.

Reply:
I am ignoring this rule in pmdrules.xml. I have restored it to ```HashMap```.